src/devices/schneider_electric.ts

```diff
@@ -1340,6 +1513,94 @@ const definitions: Definition[] = [
             commandsOnOff({'endpointNames': ['switch_sw']}),
         ],
     },
+    {
+        zigbeeModel: ['E8332DST350ZB'],
+        model: 'E8332DST350ZB',
+        vendor: 'Schneider Electric',
+        description: 'Wiser AvatarOn 2G dimmer switch',
+        extend: [
+            deviceEndpoints({endpoints: {'10': 10, '11': 11}}),
+            light({
+                endpointNames: ['10', '11'],
+                effect: false,
+                color: false,
+                powerOnBehavior: false,
+                levelConfig: {
+                    disabledFeatures: [
+                        'on_off_transition_time',
+                        'on_transition_time',
+                        'off_transition_time',
+                        'execute_if_off',
+                        'on_level',
+                        'current_level_startup',
+                    ],
+                },
+                configureReporting: true,
+            }),
+            schneiderElectricExtend.addVisaConfigurationCluster(Zcl.DataType.ENUM8),
+            schneiderElectricExtend.visaConfigIndicatorLuminanceLevel(),
+            schneiderElectricExtend.visaConfigIndicatorColor(),
+            schneiderElectricExtend.visaIndicatorMode([2, 0, 3, 1]),
+        ],
+    },
+    {
+        zigbeeModel: ['E8331SRY800ZB'],
+        model: 'E8331SRY800ZB',
+        vendor: 'Schneider Electric',
+        description: 'Wiser AvatarOn 1G onoff switch',
+        extend: [
+            deviceEndpoints({endpoints: {'10': 10}}),
+            onOff({endpointNames: ['10'], powerOnBehavior: false}),
+            schneiderElectricExtend.addVisaConfigurationCluster(Zcl.DataType.UINT8),
+            schneiderElectricExtend.visaConfigIndicatorLuminanceLevel(),
+            schneiderElectricExtend.visaConfigIndicatorColor(),
+            schneiderElectricExtend.visaIndicatorMode([0, 1, 2, 3]),
+        ],
+    },
+    {
+        zigbeeModel: ['A3N32SR800ZB_xx_C1'],
+        model: 'E8332SRY800ZB',
+        vendor: 'Schneider Electric',
+        description: 'Wiser AvatarOn 2G onoff switch',
+        extend: [
+            deviceEndpoints({endpoints: {'10': 10, '11': 11}}),
+            onOff({endpointNames: ['10', '11'], powerOnBehavior: false}),
+            schneiderElectricExtend.addVisaConfigurationCluster(Zcl.DataType.UINT8),
+            schneiderElectricExtend.visaIndicatorMode([0, 1, 2, 3]),
+        ],
+    },
+    {
+        zigbeeModel: ['E8333SRY800ZB'],
+        model: 'E8333SRY800ZB',
+        vendor: 'Schneider Electric',
+        description: 'Wiser AvatarOn 3G onoff switch',
+        extend: [
+            deviceEndpoints({endpoints: {'10': 10, '11': 11, '12': 12}}),
```

Comment:
```suggestion
            deviceEndpoints({endpoints: {'l1': 10, 'l2': 11, 'l3': 12}}),
```

Could you use a proper friendly name for all endpoint names?

Also update:

```
onOff({endpointNames: ['l1', 'l2', 'l3'], powerOnBehavior: false}),
```

Reply:
I have done this, thank you